### ISSUE TITLE
Cannot delete sequence - stuck 'uploading' or 'processing'

### ISSUE THREAD

**Issue by Nick-Tallguy** [openstreetcam/upload-scripts#83]
User=Tallguy
A sequence became 'stuck' when my computer rebooted part way through uploading. When attempting to fix this, I uploaded again, and now have 2 identical sequences, one of which is still labelled 'uploading' - https://openstreetcam.org/details/1354727/0/track-info
I've attempted to delete this sequence using the trash icon, but receive the message 'There was a problem deleting sequence!'

I have the same problem with https://openstreetcam.org/details/1357738/0/track-info which has failed to finish upload correctly, even though it appeared to finish uploading without an error. 

Sequence (only 1 picture in the sequence) https://openstreetcam.org/details/1355097/0/track-info is stuck reading 'processing' and again cannot be deleted.

**Comment by mds08011** [comment]
I am going to jump on this issue to report my own track stuck in “uploading”.  I was uploading via the scripts and had some sort of interruption which caused the error.  I am unable to delete it myself but am fine with this track being either manually stopped or deleted. 

User: mds08011
Track: https://openstreetcam.org/details/1348283/0/track-info

**Comment by pkoby** [comment]
Workaround: in the folder of photos you're trying to upload, you'll fine `count_file.txt` and `sequence_file.txt`. Copy these files into a new folder.

Now it's been awhile since I've done this, so I'm not sure which method worked, but try this:
Open `count_file.txt` and see what the highest number is (they might not be in order, so find it near the end). Copy and paste that many photos into your new folder with the text files. Try uploading that folder. Because the sequence file is in there, OSC assumes that the stuck sequence has as many photos as are in the folder now. As that now matches `count_file.txt`, it should finish and close the upload. Then you can delete the sequence and start over.

If that doesn't work, you could try editing `count_file.txt` to say `1`, copy one photo into the folder, and do as above.

Basically, I think that the error with deleting comes from it not being "finished", so you have to trick it into finishing first.

**Comment by bogdans-telenav** [comment]
I could make a command to force finish a sequence. @pkoby is right the currently the sequnce is not available  for deletion until it is processed. On the latest version of the scripts this should no longer be an issue. I am currently working on a new release with a cuple of requested features and probably will add this also.
